ocs_ci/ocs/ui/page_objects/premigration_fusion_access_ui.py

```
@@ -0,0 +1,218 @@
+
+        logger.info("Step 8: Clicking 'Create file system claim' submit button")
+        self.do_click(
+            locator=FUSION_ACCESS_STORAGE_CLUSTER_LOCATORS[
+                "submit_file_system_claim_btn"
+            ],
+            timeout=30,
+            enable_screenshot=True,
+        )
+        self.take_screenshot("file_system_claim_submitted")
+        logger.info(
+            f"File system claim '{fs_claim_name}' creation submitted successfully"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)